pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -5083,7 +5072,7 @@ func TestReconcileWithWhenExpressionsScopedToTask(t *testing.T) {
 	}
 }
 
-func TestReconcileWithWhenExpressionsScopedToTaskWitResultRefs(t *testing.T) {
+func TestReconcileWithWhenExpressionsScopedToTaskWithResultRefs(t *testing.T) {
```

Comment:
should this test name also have "scopedToTask" removed?

Reply:
@lbernick agreed, updated the test name